third_party/gfxreconstruct/framework/decode/file_processor.h

```diff
@@ -171,6 +174,10 @@ class FileProcessor
     Error                    error_state_;
     uint64_t                 bytes_read_;
 
+    // GOOGLE: [single-frame-looping] Parameters for single frame looping
+    bool     loop_single_frame_;
+    uint64_t loop_n_times_;
```

Comment:
maybe better with `loop_single_frame_count_`? also it would be clearer with some descriptions.